tests/e2e_tests/integration/05-deviceDetails.spec.ts

```diff
@@ -33,6 +33,9 @@ test.describe('Device details', () => {
     await page.click(`text=/inventory/i`);
     expect(await page.isVisible(`css=.expandedDevice >> text=Linux`)).toBeTruthy();
     expect(await page.isVisible(`css=.expandedDevice >> text=mac`)).toBeTruthy();
+    const expandedDevice = page.locator(`css=.expandedDevice`);
+    await expandedDevice.getByRole('tab', { name: /software/i }).click();
+    await page.click(`text=/software/i`);
```

Comment:
purely informative: the `page.click(`text=/software/i`);` does the same as the two lines right above (just with the risk of having a clickable text with `/software/i` in it) - but it shouldn't hurt the test here 🤞